dts/bindings/pwm/nordic,nrf-sw-pwm.yaml

```diff
@@ -0,0 +1,59 @@
+---
+title: nRF SW PWM
+version: 0.1
+
+description: >
+    This binding gives a base representation of the nRFx S/W PWM
+
+properties:
+    compatible:
+      constraint: "nordic,nrf-sw-pwm"
+
+    label:
+      type: string
+      category: required
+      description: Human readable string describing the device (used by Zephyr for API name)
+      generation: define
+
+    ch0-pin:
+      type: int
+      description: Channel 0 pin
+      category: optional
+      generation: define
+
+    ch1-pin:
+      type: int
+      description: Channel 1 pin
+      category: optional
+      generation: define
+
+    ch2-pin:
+      type: int
+      description: Channel 2 pin
+      category: optional
+      generation: define
+
+    timer-instance:
+      type: int
+      description: Timer instance to use for generating the PWM output signals
+      category: required
+      generation: define
+
+    clock-prescaler:
+      type: int
+      description: Clock prescaler for timer used for generating the PWM output signals
```

Comment:
A description of the prescaler value meaning would be nice. Something like "frequency = 16 MHz / 2^prescaler".

Reply:
Done